src/runtime/offers-flow.js

```diff
@@ -261,11 +261,11 @@ export class OffersFlow {
    * @param {!../model/client-config.ClientConfig} clientConfig
```

Comment:
nit: update description for the new param.

Reply:
Missed this, good catch!